CoronaCare/src/screens/RecordsScreen.js

```diff
@@ -31,8 +33,35 @@ const RecordsScreen = () => {
     })
   }
 
+  const onRightAction = async rowKey => {
+    const recordId = rowKey
+    Alert.alert(
+      'Warning',
+      'Are you sure you want to delete this record?',
+      [
+        {
+          text: 'Confirm',
+          onPress: async () => {
+            try{
+              deleteRecord(recordId);
+            }
+            catch (err) {
```

Comment:
What would be the condition for failing to delete the record?

Reply:
usually server side errors or transport layer errors, example, internet too slow, timeout